app/models/piece.rb

```diff
@@ -1,4 +1,8 @@
 class Piece < ActiveRecord::Base
   belongs_to :users
   belongs_to :games
+
+  def self.types
+    %w(Pawn Bishop Knight Rook Queen King)
```

Comment:
Ah okay! I was following the STI tutorial and was confused by this line, so of course I just added it in for good measure.  I've deleted this in my new commit. Thank you!